dev/tests/unit/Magento/FunctionalTestFramework/Util/Sorter/ParallelGroupSorterTest.php

```diff
@@ -54,31 +56,15 @@ public function testBasicTestsSplitByTime()
     }
 
     /**
-     * Test a sort of both tests and a suite which is larger than the given line limitation
+     * Test a sort of both tests and a suite which is larger than the given line limitation.
+     *
+     * @return void
+     * @throws FastFailException
      */
-    public function testTestsAndSuitesSplitByTime()
+    public function testTestsAndSuitesSplitByTime(): void
     {
         // mock tests for test object handler.
-        $numberOfCalls = 0;
-        $mockTest1 = AspectMock::double(
-            TestObject::class,
-            ['getEstimatedDuration' => function () use (&$numberOfCalls) {
-                $actionCount = [300, 275];
-                $result = $actionCount[$numberOfCalls];
-                $numberOfCalls++;
-
-                return $result;
-            }]
-        )->make();
-
-        $mockHandler = AspectMock::double(
-            TestObjectHandler::class,
-            ['getObject' => function () use ($mockTest1) {
-                    return $mockTest1;
-            }]
-        )->make();
-
-        AspectMock::double(TestObjectHandler::class, ['getInstance' => $mockHandler])->make();
+        $this->createMockForTest(0);
```

Comment:
$actionCount ([300, 275]) for this test is different from what's in createMockForTest() ([300, 275, 300, 275]). Please check if affects the purpose of this test.

Reply:
Thank you, @jilu1, for the great point!
I guess that I missed this one. Now it is fixed. Please, check!